blobfuse/fileapis.cpp

```diff
@@ -96,9 +96,8 @@ int azs_open(const char *path, struct fuse_file_info *fi)
             fprintf(stderr, "Failed to create file or directory on cache directory: %s, errno = %d.\n", mntPathString.c_str(),  errno);
             return -1;
         }
-        std::ofstream filestream(mntPathString, std::ofstream::binary | std::ofstream::out);
         errno = 0;
-        int fd = open(mntPath, O_WRONLY);
+        int fd = open(mntPath, O_CREAT|O_WRONLY, 0770);
```

Comment:
This would fail if the file was created already - for the attribute cache, or file cache previously. Right ? Why did you add the O_CREAT option ?